scalafix-core/shared/src/main/scala/scalafix/syntax/package.scala

```diff
@@ -74,77 +23,38 @@ package object syntax {
     }
   }
 
-  implicit class XtensionTermRef(ref: Term.Ref) {
-    def toTypeRef: Type.Ref = ref match {
-      case Term.Name(name) => Type.Name(name)
-      case Term.Select(qual: Term.Ref, Term.Name(name)) =>
-        Type.Select(qual, Type.Name(name))
-      case _ => ref.syntax.parse[Type].get.asInstanceOf[Type.Ref]
-    }
+  implicit class XtensionSymbolMirror(symbol: Symbol)(
+      implicit mirror: SemanticCtx) {
+    @deprecated("Renamed to denotation", "0.5.0")
+    def denotOpt: Option[Denotation] = denotation
+    def denotation: Option[Denotation] = mirror.denotation(symbol)
```

Comment:
I think scalafix should expose a beginner-friendly metaprogamming api. If you haven't lived and breathed compilers then I think `symbol` and `denotation` are more approachable and don't require any background to pronounce.